inference-engine/src/mkldnn_plugin/nodes/mkldnn_reference_node.cpp

```diff
@@ -22,6 +22,10 @@ MKLDNNReferenceNode::MKLDNNReferenceNode(const std::shared_ptr<ngraph::Node>& op
     }
     setType(Reference);
     setTypeStr("Reference");
+
+    if (ov::is_type<ngraph::op::v8::RandomUniform>(ngraphOp)) {
```

Comment:
We definitely need to left a comment which describes background under the condition. Please update this since it still need to be rebased on latest master

Reply:
Added comment and rebased on latest master. CI is green